backend/spec/requests/tags_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+RSpec.describe 'TagsController' do
+  describe 'POST /tags' do
+    context 'パラメーターが有効な場合' do
+      let(:valid_tag_params) { { label: Faker::Lorem.sentence(word_count: 5), color: :white, priority: 0 } }
```

Comment:
Nits

contextみたらわかるのと、
同じようなletがcontext内に存在しないため、prefixいらないと思います！

```suggestion
      let(:params) { { label: Faker::Lorem.sentence(word_count: 5), color: :white, priority: 0 } }
```